config/enums.cwt

```diff
@@ -52,6 +52,267 @@ enums = {
         very_high_ai_values_divergence #  350 #About 85% (46-98%) of characters will be at or below this value for the avarage character
         #Percentages in parentheses are the observed spans
     }
+    enum[event_themes] = {
+        abduct_scheme
+        alliance
+        bastardy
+        battle
+        befriend_scheme
+        claim_throne_scheme
+        corruption
+        crown
+        culture_change
+        death
+        default
+        diplomacy
+        diplomacy_family_focus
+        diplomacy_foreign_affairs_focus
+        diplomacy_majesty_focus
+        dread
+        dungeon
+        dynasty
+        education
+        fabricate_hook_scheme
+        faith
+        family
+        feast_activity
+        friend_relation
+        friendly
+        generic_intrigue_scheme
+        healthcare
+        hunt_activity
+        hunting
+        intrigue
+        intrigue_intimidation_focus
+        intrigue_skulduggery_focus
+        intrigue_temptation_focus
+        learning
+        learning_medicine_focus
+        learning_scholarship_focus
+        learning_theology_focus
+        love
+        lover_relation
+        marriage
+        martial
+        martial_authority_focus
+        martial_chivalry_focus
+        martial_strategy_focus
+        medicine
+        mental_break
+        mental_health
+        murder_scheme
+        party
+        pet
+        physical_health
+        pilgrimage_activity
+        pregnancy
+        prison
+        realm
+        recovery
+        rival_relation
+        romance_scheme
+        secret
+        seduce_scheme
+        seduction
+        skull
+        stewardship
+        stewardship_domain_focus
+        stewardship_duty_focus
+        stewardship_wealth_focus
+        sway_scheme
+        unfriendly
+        vassal
+        war
+        witchcraft
+    }
+    enum[event_backgrounds] = {
+        alley_day
+        alley_night
+        armory
+        army_camp
+        battlefield
+        bedchamber
+        burning_building
+        corridor_day
+        corridor_night
+        council_chamber
+        courtyard
+        docks
+        dungeon
+        farmland
+        feast
+        gallows
+        garden
+        market
+        market_east
+        market_india
+        market_tribal
+        market_west
+        physicians_study
+        sitting_room
+        study
+        tavern
+        temple
+        temple_church
+        temple_generic
+        temple_mosque
+        temple_scope
+        terrain
+        terrain_activity
+        terrain_scope
+        throne_room
+        throne_room_east
+        throne_room_india
+        throne_room_mediterranean
+        throne_room_scope
+        throne_room_tribal
+        throne_room_west
+        wilderness
+        wilderness_desert
+        wilderness_forest
+        wilderness_forest_pine
+        wilderness_mountains
+        wilderness_scope
+        wilderness_steppe
+    }
+    enum[event_environments] = {
```

Comment:
Similar to above, but: `game\gfx\portraits\environments\` I'm not sure if the folder matters for these or if it's just convention though